asyncapi/asgardeo/data_types.bal

```diff
@@ -92,6 +92,15 @@ public type LoginSuccessEvent record {
     GenericSecurityData securityData?;
 };
 
+# Login Failed event
+#
+# + eventData - Event data  
+# + securityData - Event related security data
+public type LoginFailedEvent record {
+    LoginFailedData eventData?;
```

Comment:
In a `LoginFailedEvent` notification, is `eventData` an optional field (or can this be null) ?

Reply:
this will not be null but written similar to ex existing login success event record

Reply:
Resolved in [0dc5359](https://github.com/ballerina-platform/asyncapi-triggers/pull/158/commits/0dc53596be796d9198c0557bd0a1037a6fde9d10)